packages/core-utils/src/time.js

```diff
@@ -105,29 +99,30 @@ export function formatDurationWithSeconds(seconds) {
  */
 export function formatTime(ms, options) {
   return format(
-    ms + (options && options.offset ? options.offset : 0),
-    options && options.format ? options.format : OTP_API_TIME_FORMAT
+    ms + (options?.offset || 0),
+    options?.format || OTP_API_TIME_FORMAT
   );
 }
 
 /**
  * Formats a seconds after midnight value for display in narrative
  * @param  {number} seconds  time since midnight in seconds
- * @param  {string} timeFormat  A valid moment.js time format
+ * @param  {string} timeFormat  A valid date-fns time format
  * @return {string}                   formatted text representation
  */
 export function formatSecondsAfterMidnight(seconds, timeFormat) {
-  return format(add(startOfDay(new Date()), { seconds }), timeFormat);
+  const time = add(startOfDay(new Date()), { seconds });
+  return format(time, timeFormat);
 }
 
 /**
- * Get the timezone name that is set for the user that is currently looking at
- * this website. Use a bit of hackery to force a specific timezone if in a
- * test environment.
+ * Uses Intl.DateTimeFormat() api to get the user's time zone. In a test
+ * environment, pulls timezone information from an env variable. Default to
+ * GMT+0 if the Intl API is unavailable.
  */
 export function getUserTimezone() {
   if (process.env.NODE_ENV === "test") return process.env.TZ;
-  return Intl.DateTimeFormat().resolvedOptions().timeZone;
+  return Intl?.DateTimeFormat().resolvedOptions().timeZone || "Etc/Greenwich";
```

Comment:
One final item: instead of defaulting to GMT+0, could we make that an optional arg to this method? E.g.:
```javascript
export function getUserTimezone(fallbackTimeZone = "Etc/Greenwich") {
```

Reply:
Good call! Changed in 8d963b4ec3e2a81a4a3e709c5efbafb94d8d098c